examples/cms-buttercms/.eslintrc

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "next/core-web-vitals",
+    "rules": {
+        // Other rules
+        "@next/next/no-img-element": "off"
```

Comment:
> If i remember correctly, this author card component was originally set up with <Image />, but was changed to a <img> because of the need for people to choose author profile images from whatever domain they wanted, as people have the option to set their profile image in butter to any image (e.g., outside of our cms).

We should still keep image - any other way to work around this?

Reply:
We decided to just pull the author profile image out as it was not necessary at this time, so this rule has been ignored and the supplementary .eslintrc file has been deleted to avoid conflicts